core/replay/replay.py

```diff
@@ -200,6 +208,25 @@ def main():
 
         logger.info(f'Exported system tables to {g_config["replay_output"]}')
 
+    # uploading replay logs to s3
+
+    bucket = bucket_dict(g_config["workload_location"])
```

Comment:
similarly , if the workload location is not S3, this will fail.
Remember for Replay we were testing a 48 query workload from local directory ?

Reply:
agree what @DevanathanSabapathy1 
We need error handling for this case too